### ISSUE TITLE
Corner resize handles

### ISSUE THREAD

**Issue by Slapbox** [aeagle/react-spaces#127]
**Is your feature request related to a problem? Please describe.**
Alternative of resizing one edge and then the other is tedious

**Describe the solution you'd like**
It would be great to be able to resize from the corners instead

**Comment by aeagle** [comment]
PR #135

**Comment by aeagle** [comment]
Hi @Slapbox.

This functionality has now been released in `v0.3.8`. 

The `resizable` property on a `<Positioned />` space can be as granular as you like, e.g.:

```
<Positioned resizable={[ ResizeType.Right, ResizeType.BottomRight ]} ... />
```

or as a shortcut for all sides + corners as resizable:

```
<Positioned resizable={[ ResizeType.All ]} />
```

**Comment by Slapbox** [comment]
Very cool! I can't wait for crunch time to end here so I can give it a try.

Thanks for your continued excellent work @aeagle!
